### PR TITLE
CB-8681 CB-3257 Adding GCS support for telemetry Logging

### PR DESCRIPTION
**Pull request by cegganesh84**
1. This is required for creating CDP cli for environment creation.
2. The actual logging is yet to be done.

./gradlew build

### CODE REVIEW

File: freeipa/src/main/java/com/sequenceiq/freeipa/converter/telemetry/TelemetryConverter.java [hunk 1]
```diff
@@ -81,6 +82,10 @@ private Logging createLoggingFromRequest(LoggingRequest loggingRequest) {
                 adlsGen2Params.setSecure(adlsGen2FromRequest.isSecure());
                 adlsGen2Params.setManagedIdentity(adlsGen2FromRequest.getManagedIdentity());
                 logging.setAdlsGen2(adlsGen2Params);
+            } else if (loggingRequest.getGcs() != null) {
```
Comment (doktoric) [inline freeipa/src/main/java/com/sequenceiq/freeipa/converter/telemetry/TelemetryConverter.java:85]: the unit test part should be also added

File: datalake/src/main/java/com/sequenceiq/datalake/service/sdx/CloudStorageManifester.java [hunk 2]
```diff
@@ -97,6 +97,8 @@ private void addLogIdentity(CloudStorageRequest cloudStorageRequest,
                 log.setS3(logging.getS3());
             } else if (logging.getAdlsGen2() != null) {
                 log.setAdlsGen2(logging.getAdlsGen2());
+            } else if (logging.getGcs() != null) {
```
Comment (doktoric) [inline datalake/src/main/java/com/sequenceiq/datalake/service/sdx/CloudStorageManifester.java:100]: unit test will be required probably

**Review by doktoric — CHANGES_REQUESTED** [review]
comments were added

**Comment by cegganesh84** [comment]
Thanks, @doktoric, added the unit test, and was able to fix a couple of bugs I introduced!

**Review by doktoric — APPROVED** [review]

**Pull request merged**